src/views/tutorial-view/index.tsx

```diff
@@ -109,10 +112,15 @@ export default function TutorialView({
             readTime,
             edition,
             productsUsed,
-            isInteractive: Boolean(handsOnLab),
-            hasVideo: Boolean(video),
+            isInteractive,
+            hasVideo,
           }}
         />
+        {hasVideo && video.id && !video.videoInline && (
```

Comment:
Question/nit - are `video.id` and `!video.videoInline` a part of determining `hasVideo`? If so, is it worth moving them up to the `hasVideo` declaration? No action required for this on this PR, mainly asking for understanding. 😊

Reply:
Great Q - the existing check for whether to show the "Video" badge in `TutorialMeta` looked only at whether `video` was truthy, without validating there's a `video.id`. To match the behaviour in `learn`, I've ported over that same logic.

It is kinda odd that for the badge, we validate only that `video` is truthy; while for the display of `VideoEmbed` after metadata, we also validate that `video.id` is truthy 🤔 Seems like if there's any possibility that `video` is truthy but `video.id` is undefined, maybe we should validate / throw an error in tutorial content validation? It's possible we already do that so the `video.id` check is a bit redundant? cc @kendallstrautman

Reply:
Yeah yeah, I agree with all you've said here! This seems like something that should be the burden of the content validation. Thank you for the context! 👍🏻

Reply:
@zchsh sorry late to the party here. In previous Learn implementation `videoId` was its own property. Here we've adjusted so that `video` is an object with extra properties. If `id` is undefined, `video` should be undefined. `videoInline` indicates whether the video is embedded within the mdx content and not at the top of the page here. I hope that helps to clarify!